ebp/baseebp.go

```diff
@@ -138,3 +138,16 @@ func (ebp *baseEbp) SetIsEmpty(value bool) {
 		ebp.DataFieldLength = 1
 	}
 }
+
+// StreamSyncSignal checks the Grouping ID field and return the Stream Sync Signal
```

Comment:
just a nitpick: return -> returns
Also thats a great description for what we are doing in this function, 
Can we also please add a desc of what a stream sync signal means when we receive it from the tx?

Reply:
also, is the stream sync signal be something thats going to be present in the comcast and cablelabs ebp? I ask as if its only going to be present in either cablelabs or comcast EBP, then maybe we can put it only in that specific EBP code.

Reply:
I'm not exactly sure if comcast EBP will ever implement it, since it is not being used by any of the Transcoder vendors now. But Paul E. mentioned that VACT is working on auditing configurations to standardize them in order to implement Stream Sync, so I just put this part of the implementation in place towards that direction.